main.cpp

```diff
@@ -12,76 +12,203 @@ What you create in this project will be used as the basis of Project 5 in the co
+        brightness
+        amount of power consumed.
+        pixels
+        width in cm
+        height in cm
+
+    the Display's brightness can be represented with a Primitive, such as a double. 
+
+    The amount of power consumed can also be represented with a Primitive, such as a float or integer (i.e. 250mWa)
     
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
+    The 'pixels' property must be represented with an array of Pixel instances, as the screen has more than 1 row of pixels.
+        Arrays have not been discussed and can't be used in this project.
+        Instead, we can use an Integer primitive to store the Number of Pixels:
+
+    Display:
+        Number of Pixels
+        Amount of Power consumed (milliwatt-hours)
+        Brightness
+        width in cm
+        height in cm
+************************
+0) move 5-9 to between your Thing 4 and Thing 10.
+
+1) Fill in #5 - #9 with plain-english UDTs for the 5 properties you created for UDT #10
+    example: 
+        If #10's first property was 'Engine', then your `Thing 5)` will be `Engine` and 
+        you will need to provide 5 properties and 3 member functions of that Engine object in plain English
 */
+
+/*
+Thing 1) Hotel
+5 properties:
+    1) Number of rooms
+    2) Number of guests
+    3) Gross revenue
+    4) Overhead costs per anum
+    5) Number of employees
+3 things it can do:
+    1) Book in guests
+    2) Clean rooms
+    3) Valet service
+ */
+
+/*
+Thing 2) Printer
+5 properties:
+    1) Ink levels 
+    2) Electrcity consumption rate
+    3) Paper level
+    4) Maximum resolution
+    5) Brand name
+3 things it can do:
+    1) Print documents
+    2) Load print jobs
+    3) Scan documents
+ */
+
+/*
+Thing 3) Oven
+5 properties:
+    1) Maximum temperature
+    2) Number of trays
+    3) Model ID
+    4) Energy consumption rating
+    5) Cost
+3 things it can do:
+    1) Cook food
+    2) Consume electrictiy
+    3) Power filiments
+ */
+
+/*
+Thing 4) Music studio
+5 properties:
+    1) Number of microphones
+    2) Available instruments
+    3) Available outboard equipment
+    4) Number of engineers
+    5) Cost per hour
+3 things it can do:
+    1) Recording musicians
+    2) Mix audio
+    3) Master audio
+ */
+
+/*
+Thing 5) Wheels
+5 properties:
+    1) tyres
+    2) hubcaps
+    3) bolts
```

Comment:
Tyres, Hubcaps, bolts.   These are THINGS, not properties.
You need to pick properties that can be represented with `int float double bool char std::string`
- [ ] revise